test/controllers/config_reports_controller_test.rb

```diff
@@ -114,4 +141,10 @@ def test_destroy
   def create_a_report
     @report = ConfigReport.import(read_json_fixture('reports/empty.json'))
   end
+
+  def create_a_report_with_origin(origin)
+    report = FactoryBot.create(:config_report)
+    report.origin = origin
+    report.save!
```

Comment:
AFAIK you can use
```suggestion
    FactoryBot.create(:config_report, origin: origin)
```

But that generates a DB query, which is slower. If you don't need the entry persisted you can also use `FactoryBot.build(:config_report, origin: origin)`. Though with the controller it's probably needed to persist it.

Also note that it's small enough to not need a separate method.

Reply:
I've just tested it and it needs to be persisted - so, I'm not gonna make a separate method & do the following?
```
report = FactoryBot.create(:config_report, origin: 'Ansible')
report.save!
```

Reply:
I thought `create()` already persisted. That's the difference to `build()`.

Reply:
You're right! Didn't know about the difference tbh.